FilterViewController.h

```diff
@@ -12,5 +12,6 @@
 @interface FilterViewController : UITableViewController<MHCollapibleViewManagerDelegate>
 
 @property (strong, nonatomic) NSMutableArray *managerArray;
+@property IBOutlet UIButton *clearButton;
```

Comment:
I see, good to know. It has already been removed in a later PR. I accessed the button calls in a method and didn't want to keep an instance of them.